linux/keyman-config/Makefile

```diff
@@ -8,17 +8,20 @@ langtags:
 
 install: # run as sudo
 	pip3 install qrcode sentry-sdk
+	# eventually change this to: pip3 install .
 	python3 setup.py install
 	# install icons
 	mkdir -p /usr/local/share/keyman/icons
 	cp keyman_config/icons/* /usr/local/share/keyman/icons
 	# install man pages
 	mkdir -p /usr/local/share/man/man1
-	cp debian/man/*.1 /usr/local/share/man/man1
+	cp ../../debian/man/*.1 /usr/local/share/man/man1
```

Comment:
hrm … - okay, but ../../debian/man isn't in .gitignore, is it?

Reply:
> hrm … - okay, but ../../debian/man isn't in .gitignore, is it? 

Hmm, you're right. It's ignored on my machine, but why?